[2024-04-23T11:21:24.028+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-16T00:00:00+00:00 [queued]>
[2024-04-23T11:21:24.033+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-16T00:00:00+00:00 [queued]>
[2024-04-23T11:21:24.034+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-23T11:21:24.045+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): gen_dates> on 2024-03-16 00:00:00+00:00
[2024-04-23T11:21:24.050+0000] {standard_task_runner.py:60} INFO - Started process 76496 to run task
[2024-04-23T11:21:24.052+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_eq', 'gen_dates', 'scheduled__2024-03-16T00:00:00+00:00', '--job-id', '253', '--raw', '--subdir', 'DAGS_FOLDER/kbs/custom_dags/pipeline_eq.py', '--cfg-path', '/tmp/tmp79x0u_eb']
[2024-04-23T11:21:24.053+0000] {standard_task_runner.py:88} INFO - Job 253: Subtask gen_dates
[2024-04-23T11:21:24.077+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-16T00:00:00+00:00 [running]> on host datalab-yasmine
[2024-04-23T11:21:24.117+0000] {logging_mixin.py:188} WARNING - /home/yasmine/.local/share/virtualenvs/knowledge-base-siaap-WvBZnYs-/lib/python3.8/site-packages/airflow/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-04-23T11:21:24.266+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_eq' AIRFLOW_CTX_TASK_ID='gen_dates' AIRFLOW_CTX_EXECUTION_DATE='2024-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-16T00:00:00+00:00'
[2024-04-23T11:21:24.267+0000] {rw.py:84} INFO - Starting to save files into Minio
[2024-04-23T11:21:26.442+0000] {python.py:202} INFO - Done. Returned value was: dates_20240316
[2024-04-23T11:21:26.456+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=pipeline_eq, task_id=gen_dates, execution_date=20240316T000000, start_date=20240423T112124, end_date=20240423T112126
[2024-04-23T11:21:26.474+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T11:21:26.495+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-23T11:24:49.124+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-16T00:00:00+00:00 [queued]>
[2024-04-23T11:24:49.129+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-16T00:00:00+00:00 [queued]>
[2024-04-23T11:24:49.129+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-23T11:24:49.140+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): gen_dates> on 2024-03-16 00:00:00+00:00
[2024-04-23T11:24:49.145+0000] {standard_task_runner.py:60} INFO - Started process 81576 to run task
[2024-04-23T11:24:49.147+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_eq', 'gen_dates', 'scheduled__2024-03-16T00:00:00+00:00', '--job-id', '275', '--raw', '--subdir', 'DAGS_FOLDER/kbs/custom_dags/pipeline_eq.py', '--cfg-path', '/tmp/tmp8e0zy_7o']
[2024-04-23T11:24:49.147+0000] {standard_task_runner.py:88} INFO - Job 275: Subtask gen_dates
[2024-04-23T11:24:49.171+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-16T00:00:00+00:00 [running]> on host datalab-yasmine
[2024-04-23T11:24:49.201+0000] {logging_mixin.py:188} WARNING - /home/yasmine/.local/share/virtualenvs/knowledge-base-siaap-WvBZnYs-/lib/python3.8/site-packages/airflow/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-04-23T11:24:49.317+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_eq' AIRFLOW_CTX_TASK_ID='gen_dates' AIRFLOW_CTX_EXECUTION_DATE='2024-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-16T00:00:00+00:00'
[2024-04-23T11:24:49.318+0000] {logging_mixin.py:188} INFO - {'start': '2024-03-16 00:00:00+00:00', 'end': '2024-04-23 11:24:49'}
[2024-04-23T11:24:49.318+0000] {rw.py:84} INFO - Starting to save files into Minio
[2024-04-23T11:24:51.333+0000] {python.py:202} INFO - Done. Returned value was: dates_20240316
[2024-04-23T11:24:51.347+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=pipeline_eq, task_id=gen_dates, execution_date=20240316T000000, start_date=20240423T112449, end_date=20240423T112451
[2024-04-23T11:24:51.373+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T11:24:51.385+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
